### PR TITLE
Adding footer to the pages

### PR DESCRIPTION
**Pull request by Iseost**
Added footer to the pages that is created. Needs media query and a prfil image

### CODE REVIEW

File: pages/blog.html [hunk 1]
```diff
@@ -244,6 +244,141 @@ <h2 class="font-semibold text-3xl">Title Here</h2>
...
+
+      <!--PP, TC and ©-->
+      <div class="flex justify-center mt-32">
+        <a class="underline pr-6" href="#">Privacy Policy</a>
+        <a class="underline pl-6" href="#">Terms and Conditions</a>
+      </div>
+      <div class="flex justify-center p-6">&copy;Musikkforandrerliv</div>
+    </footer>
+    <!--Footer end-->
     </main>
```
Comment (EliNygard) [inline pages/blog.html:382]: Close main before footer

File: pages/support-page.html [hunk 2]
```diff
@@ -320,6 +320,141 @@ <h3 class="text-2xl font-semibold">Subscribe to our newsletter</h3>
...
+  
+        <!--PP, TC and ©-->
+        <div class="flex justify-center mt-32">
+          <a class="underline pr-6" href="#">Privacy Policy</a>
+          <a class="underline pl-6" href="#">Terms and Conditions</a>
+        </div>
+        <div class="flex justify-center p-6">&copy;Musikkforandrerliv</div>
+      </footer>
+      <!--Footer end-->
     </main>
```
Comment (EliNygard) [inline pages/support-page.html:458]: Close main before footer

**Review by EliNygard — CHANGES_REQUESTED** [review]
Double check that the main element is closed before the footer element on all pages.

**Comment by EliNygard** [comment]
Some pages do not have a main tag. Maybe go over with your team and check all pages :)

File: pages/gallery-page.html [hunk 3]
```diff
@@ -8,72 +8,306 @@
...
+        <button
+          id="closeSearchBarDrawer"
+          class="search-drawer_close text-sm pr-5"
+        >
+          Close
+        </button>
+        <!-- JS intention: On click, add -translate-y-full to section id="search-drawer-from-header" and remove translate-y-0 -->
+      </div>
+    </section>
+    </main>
```
Comment (EliNygard) [inline pages/gallery-page.html:175]: I can't find the opening tag for main

File: pages/about-page.html [hunk 4]
```diff
@@ -1,75 +1,309 @@
...
+        >
+          Close
+        </button>
+        <!-- JS intention: On click, add -translate-y-full to section id="search-drawer-from-header" and remove translate-y-0 -->
+      </div>
     </section>
-</body>
-</html>
\ No newline at end of file
+    </main>
```
Comment (EliNygard) [inline pages/about-page.html:171]: I can't find a opening tag for main. No need to close it if the tag is never used.

**Review by EliNygard — CHANGES_REQUESTED** [review]
Two comments, the rest looks perfect :)

**Comment by EliNygard** [comment]
Hi again. Many pages have been updated since this PR and resulted in conflicts on 8 pages. Is it possible for you to create a new PR with a branch that is up to date with main branch?

**Comment by Iseost** [comment]
I will fix it asap :)

man. 8. apr. 2024 kl. 10:02 skrev Eli Nygård ***@***.***>:

**Comment by EliNygard** [comment]
Great :) What is status? Should we close this PR?

**Comment by annechh** [comment]
This issue can be closed. There has been opened a new issue for this

**Comment by EliNygard** [comment]
Wonderful. I will close it.

**Pull request closed without merging**